functions.Rmd

```diff
@@ -14,6 +14,24 @@ row_adder()
 permutation()
 ```
 
+## Anonymous functions
```

Comment:
Is it clear to the audience what "anonymous" means?

Reply:
Added a bit more context